packages/swap/src/ingress-egress-tracking/__tests__/index.test.ts

```diff
@@ -51,7 +52,7 @@ describe('ingress-egress-tracking', () => {
         'deposit:Ethereum:0x1234',
         JSON.stringify({
           amount: '0x9000',
-          asset: 'FLIP',
+          asset: { asset: 'FLIP', chain: 'Ethereum' },
```

Comment:
Not sure why this was needed here. Does it mean that we also have to adjust the deposit monitor tool to insert the data in this format?

Reply:
No this is the "new" format. It's been this way for a long time, but we never removed the old logic